pymc3/tests/test_distributions.py

```diff
@@ -1371,28 +1398,38 @@ def test_binomial(self):
     @pytest.mark.xfail(
         condition=(SCIPY_VERSION < parse("1.4.0")), reason="betabinom is new in Scipy 1.4.0"
     )
-    def test_beta_binomial(self):
-        self.checkd(
+    def test_beta_binomial_logcdf(self):
+        self.check_logcdf(
             BetaBinomial,
             Nat,
             {"alpha": Rplus, "beta": Rplus, "n": NatSmall},
+            lambda value, alpha, beta, n: sp.betabinom.logcdf(value, a=alpha, b=beta, n=n),
+            n_samples=-1,
         )
-        self.check_logp(
+
+    def test_beta_binomial(self):
+        self.checkd(
             BetaBinomial,
             Nat,
             {"alpha": Rplus, "beta": Rplus, "n": NatSmall},
```

Comment:
This one should be tested with n_samples=-1, but you have to change it in `check_int_to_1`. Maybe let's add the optional argument `n_samples=10` to `check_int_to_1`, and then you can temporarily call this test via:

```python
    def test_beta_binomial(self):
        self.checkd(
            BetaBinomial,
            Nat,
            {"alpha": Rplus, "beta": Rplus, "n": NatSmall},
            checks=(lambda *args: self.check_int_to_1(*args, n_samples=-1),
       )

Reply:
This one failed in here: https://github.com/pymc-devs/pymc3/pull/4497/checks?check_run_id=2021551266

In checkd:
```E           AssertionError: 
E           Arrays are not almost equal to 7 decimals
E           {'alpha': array(0.1, dtype=float32), 'beta': array(0.01, dtype=float32), 'n': array(5), 'value': array(0, dtype=int16)}
E            ACTUAL: 0.9999997497252575
E            DESIRED: 1
```

Reply:
Xfail added

Reply:
Xfailed as expected